components/tidb_qe/src/error.rs

```diff
@@ -13,17 +13,17 @@ pub enum EvaluateError {
     Custom { code: i32, msg: String },
 
     #[fail(display = "{}", _0)]
-    Unknown(String),
+    Other(String),
 }
 
 impl EvaluateError {
     /// Returns the error code.
     pub fn code(&self) -> i32 {
         match self {
             // TODO: We should assign our own error code
-            EvaluateError::MaxExecuteTimeExceeded => 1105,
+            EvaluateError::MaxExecuteTimeExceeded => 8501,
```

Comment:
Note: This code is self-defined. It is changed to `9007` in later commit to better fit TiDB's definition:

```go
	// TiDB self-defined errors.
	ErrMemExceedThreshold = 8001
	ErrForUpdateCantRetry = 8002
	ErrAdminCheckTable    = 8003

	// TiKV/PD errors.
	ErrPDServerTimeout    = 9001
	ErrTiKVServerTimeout  = 9002
	ErrTiKVServerBusy     = 9003
	ErrResolveLockTimeout = 9004
	ErrRegionUnavailable  = 9005
	ErrGCTooEarly         = 9006
```